pkgs/top-level/all-packages.nix

```diff
@@ -37132,6 +37132,8 @@ with pkgs;
 
   masari = callPackage ../applications/blockchains/masari { };
 
+  mev-boost = callPackage ../applications/blockchains/mev-boost { };
```

Comment:
If possible, this should be removed and the package should then be moved to the `pkgs/by-name/so/some-example/package.nix` hierarchy, as it is the preferred way of adding new packages.

Reply:
Thanks for the review. I have implemented all of your suggested changes